### DISCUSSION TITLE
Custom Buttons

### DISCUSSION THREAD

**Discussion by hanan-commits-work** [iTwin/itwinjs-core#3050] [Q&A]
Since ButtonType is removed, is there any way to make custom buttons using the ui-react, apart from cta, borderless, etc?

**Comment by bsteinbk** [comment]
@hanan-commits-work I am not sure what you mean by 'custom` buttons, but to see how we internally convert from ButtonType to props for an itwinui Button see our [core-react Button](https://github.com/iTwin/itwinjs-core/blob/b7b9b5587412b3365041b65d048baf1e1236034e/ui/core-react/src/core-react/button/Button.tsx#L53).
